### PR TITLE
Redirect to treasury

### PR DESCRIPTION
**Pull request by Darruma**

### CODE REVIEW

File: rewards/emission_handlers.py [hunk 1]
```diff
@@ -24,5 +24,12 @@ def unclaimed_rewards_handler(amount: Decimal, token: str, sett: str, block: int
 def ibbtc_peak_handler(amount: Decimal, token: str, sett: str, block: int) -> RewardsList:
     console.log("Distributing {} of {} to ibbtc multisig".format(amount, token))
     rewards = RewardsList()
-    rewards.increase_user_rewards(IBBTC_MULTISIG, token, amount)
+    rewards.increase_user_rewards(addresses.IBBTC_MULTISIG, token, amount)
+    return rewards
+
+
+def treasury_handler(amount: Decimal, token: str, sett: str, block: int) -> RewardsList:
```
Comment (SHAKOTN) [inline rewards/emission_handlers.py:31]: Think this new code should be covered with tests

File: rewards/emission_handlers.py [hunk 2]
```diff
@@ -24,5 +24,12 @@ def unclaimed_rewards_handler(amount: Decimal, token: str, sett: str, block: int
...
+    console.log("Distributing {} of {} to treasruy multisig".format(amount, token))
```
Comment (SHAKOTN) [inline rewards/emission_handlers.py:32]: `treasruy` typo

Comment (SHAKOTN) [reply rewards/emission_handlers.py:32]: Also, why not using f-string for formatting?

**Review by mitche50 — APPROVED** [review]

**Review by SHAKOTN — APPROVED** [review]
lgtm

**Pull request merged**